Week2/todo_app/program.js

```diff
@@ -62,11 +120,64 @@ dbConnection.connect(function(err) {
     console.log('connected as id ' + dbConnection.threadId);
 
     const todoModel = new TodoModel(dbConnection);
-    todoModel.load(function(err, todoItems) {
-        if(err) {
+    todoModel.load(function (err, todoItems) {
+        if (err) {
+            console.log("error loading TODO items:", err);
+        }
+
+        console.log("existing todo items:", todoItems);
+    });
+
+
+    // // create
+    todoModel.create(2, "create anything", function (err, todoItems) {
```

Comment:
We'd like our app to be interactive - get input from users and function really as an app. Can you think about how to do that?